### PR TITLE
Add Circle CI Configuration

### PR DESCRIPTION
**Pull request by tylerwilding-wk**
Adds CircleCI Configuration as well as some new scripts to test documentation and license headers during the build.

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .circleci/clojars/profiles.clj [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{:auth
+ {:repository-auth
+  {#"https//repo.clojars.org"
+   {:username #=(eval (System/getenv "CLOJARS_USER"))
```
[the remark this answers is not in the document]
Comment (tylerwilding-wk) [reply .circleci/clojars/profiles.clj]: This is to push to clojars

File: .circleci/scripts/verify-dockerfile.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Checks if either the `docker build ...` command exited with '0' or
+# if the dockerfile ends with FROM scratch, no image will be generated which
+# is causes exit code '1', however this is valid from our internal CI's perspective
```
[the remark this answers is not in the document]
Comment (tylerwilding-wk) [reply .circleci/scripts/verify-dockerfile.sh]: I consider it worth mentioning for our own internal documentation purposes: someone at Workiva wondering why we have such a script or if a third-party is trying to figure out why the dockerfile step is failing.  All this says is that our internal CI is running `docker build` on the included file, I don't think thats much of a secret when we have a file called workivabuild.Dockerfile.

**Comment by danielharasymiw-wf** [comment]
+1

**Comment by tylerwilding-wk** [comment]
@Workiva/release-management-p

**Pull request merged**